Rubberduck.Deployment/BuildRegistryScript.ps1

```diff
@@ -95,9 +95,17 @@ try
 				if (Test-Path -Path $regFile -PathType Leaf)
 				{
 					$datetime = Get-Date;
-					& reg.exe import $regFile;
+					if ([Environment]::Is64BitOperatingSystem)
+					{
+						& reg.exe import $regFile /reg:32;
+						& reg.exe import $regFile /reg:64;
+					}
+					else 
+					{
+						& reg.exe import $regFile /reg:32;					
+					}
 					& reg.exe import ($dir + "\RubberduckAddinRegistry.reg");
-					Move-Item -Path $regFile -Destination ($regFile + ".imported_" + $datetime.ToUniversalTime().ToString("yyyyMMddhhmmss") + ".txt" )
+					Move-Item -Path $regFile -Destination ($regFile + ".imported_" + $datetime.ToUniversalTime().ToString("yyyyMMddHHmmss") + ".txt" )
```

Comment:
Small thing, hope you don't mind. Having hours set to am/pm was disordering the files if ordered by name. Also, European habit, I'm so used to my 24h timing.

Reply:
Oversight on my part. Good catch - yes I meant to use 24 hours clock.